pkg/network/node/node.go

```diff
@@ -144,36 +144,17 @@ func New(c *OsdnNodeConfig) (network.NodeInterface, error) {
 		return nil, nil
 	}
 
+	if useConnTrack && c.ProxyMode != componentconfig.ProxyModeIPTables {
```

Comment:
I feel like it's better to always log the "Initializing SDN" message... can you move things around so that happens before this? (And why did you move this before `setNodeIP`? Why does that matter?)

Reply:
Agree, it is always good to log this msg. 
This proxy check doesn't depend on computing hostname/selfIP, so I just moved that logic a bit up in this method.